src/System.Xml.XDocument/tests/events/System.Xml.XDocument.Events.Tests.csproj

```diff
@@ -9,7 +9,6 @@
     <AssemblyName>System.Xml.XDocument.Events.Tests</AssemblyName>
     <SignAssembly>false</SignAssembly>
     <ProjectGuid>{C560E194-5B14-4112-ABC6-3208491E53E6}</ProjectGuid>
-    <TestCategories>OuterLoop</TestCategories>
```

Comment:
We should continue supporting marking tests projects as OuterLoop and then completely skip running them when we are only doing InnerLoop test runs.